.github/workflows/maestro_deploy-tests.yaml

```diff
@@ -9,8 +9,19 @@ on:
 jobs:
   deploy-tests:
     runs-on: ubuntu-latest
+    env:
+      IN_GITHUB_ACTION: 1
```

Comment:
IIUC you'll need to add DEPLOY_KUBERNETES_TEST and DEPLOY_DOCKER_TEST for the tests to actually run. In addition this won't run the tests in `tests/cli/test_commands.py` just `tests/integration/deploys/test_deploy.py`